<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceServiceDialog</class>
 <widget class="QDialog" name="DeviceServiceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string>Statuslabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelError">
     <property name="text">
      <string>Errorlabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidgetDevices"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
